proc/test_thread_rand.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-tests: random lifespan threads test
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <errno.h>
+#include <sys/threads.h>
+
+#include "unity_fixture.h"
+
+
+typedef struct {
+	unsigned int time;
+	unsigned int id;
+} thread_params_t;
+
+
+typedef struct {
+	thread_params_t params;
+	char stack[512] __attribute__((aligned(8))); /* stack needs to be aligned to 8 */
+} thread_self_t;
+
+
+typedef struct {
+	unsigned int fin;
+	handle_t fin_lock;
```

Comment:
Please change this name to word without `_` to be inline with our coding standards.

Reply:
Yes, that name is not within our coding standards. I will fix that.